### PR TITLE
ExceptionHandler 中 Exception 部分返回 body 参数增加 html 转义

### PR DESCRIPTION
**Pull request by DarrenLuo**
relation #10640

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

在 ExceptionHandler 中，返回的数据如果存在 html 标签，则可能导致前端出现一些特殊情况，所以对于未知的 Exception 返回 body 部分增加 html 转义

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java [hunk 1]
```diff
@@ -37,28 +38,30 @@
...
         return ResponseEntity.status(HttpStatus.FORBIDDEN).body(e.getErrMsg());
     }
-    
+
     @ExceptionHandler(IllegalArgumentException.class)
     private ResponseEntity<String> handleIllegalArgumentException(IllegalArgumentException e) {
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(ExceptionUtil.getAllExceptionMsg(e));
     }
-    
+
```
Comment (KomachiSion) [inline console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java:54]: Please don't change the indent. Use nacos code style to reformat codes.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java (hunk 1 above)
Comment (DarrenLuo) [reply console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java]: Handled

File: console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java [hunk 2]
```diff
@@ -57,8 +58,9 @@ private ResponseEntity<Object> handleException(HttpServletRequest request, Excep
         LOGGER.error("CONSOLE {}", uri, e);
         if (uri.contains(Commons.NACOS_SERVER_VERSION_V2)) {
             return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
-                    .body(RestResultUtils.failed(ExceptionUtil.getAllExceptionMsg(e)));
+                    .body(RestResultUtils.failed(HtmlUtils.htmlEscape(ExceptionUtil.getAllExceptionMsg(e), "utf-8")));
         }
-        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(ExceptionUtil.getAllExceptionMsg(e));
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR)
+                .body(HtmlUtils.htmlEscape(ExceptionUtil.getAllExceptionMsg(e), "utf-8"));
```
Comment (KomachiSion) [inline console/src/main/java/com/alibaba/nacos/console/exception/ConsoleExceptionHandler.java:64]: Why directly set utf-8？

**Comment by DarrenLuo** [comment]
这里是提供给 HtmlUtils.htmlEscape 方法使用的，如果不设置的话默认是 iso-8859-1，会导致西文之外的字符集乱码
1687942636139_48DA8745-A755-44f0-BFD1-E39AF4AD7CA7

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**